Library/StripeTypes.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import KsApi
+
+public protocol StripeSCARequiring {
```

Comment:
This protocol helps to merge the signals in `PledgeViewModel` for simplification.